app/graphql/types/hmis_schema/project.rb

```diff
@@ -12,13 +12,14 @@ class HmisSchema::Project < Types::BaseObject
     field :id, ID, null: false
     field :name, String, null: false
     field :project_type, Types::HmisSchema::ProjectType, null: false
+    field :organization, Types::HmisSchema::Organization, null: true
 
-    def self.projects(context, project_types)
-      viewable_projects = GrdaWarehouse::Hud::Project.viewable_by(context[:current_user])
-      viewable_projects = viewable_projects.with_project_type(project_types) if project_types.present?
+    def name
+      object.name(context[:current_user])
+    end
```

Comment:
So we should _not_ make this call to `name(..)` anymore, based on @eanders other comments. Could be replaced with the below suggestion, or just replace line 13 with `field :ProjectName, String, null: false`, not sure what we want to do here. @awisema 

```suggestion
    def name
      object.ProjectName
    end
```

Reply:
@gigxz Are we using TitleCase for GraphQL field names? Usually I'd declare them using snake_case and then ruby-graphql converts them to camelCase. Is that fine, or do we need to use a different capitalization convention? We can, I just want to be clear about it.

Reply:
Oh I don't know about the casing for GraphQL field names. Do whatever is the norm! But on the active record model, the name field is called `ProjectName` so that's how you want to access it. I might be misunderstanding what this code is doing.

Reply:
@gigxz I think we were using `name(user)` to handle confidential project names, so that it would replace the name with `Confidential Project` if it was confidential. If we don't want to use the `name` method, but **do** want to hande confidential names, we could probably do:

```ruby
    def name
      object.confidential ? 'Confidential Project' : object.ProjectName
    end
```

But I'm not totally sure how we want to handle it either. Maybe we should just use `object.ProjectName` for now?

Reply:
Sorry my comment was unclear. I think because we are not using the warehouse permissions, we should just ignore confidentiality status for now, and always return the real name of the project.

Reply:
Gotcha, I'll make the adjustment. Should we leave the `excludeConfidential` argument for now, or should I remove that as well?